pkg/artifact/consts.go

```diff
@@ -3,5 +3,5 @@ package artifact
 const (
 	// UnknownConfigMediaType is the default mediaType used when no
 	// config media type is specified.
-	UnknownConfigMediaType = "application/vnd.unknown.config.v1+json"
```

Comment:
This got a little more complex.
We agreed to pull the encoding off, when the object is null.
When the object is set, it would have the encoding. `+json` seems like a good default.